app/index.js

```diff
@@ -1,8 +1,11 @@
 const server = require('./server')
+const db = require('./data')
 
 const init = async () => {
   await server.start()
   console.log('Server running on %s', server.info.uri)
+  const examples = await db.example.findAll()
+  console.table(examples)
```

Comment:
Added this as a demo for how to query database tables using Sequelize in your app.  Very basic `SELECT* FROM examples;` equivalent here.

Feel free to delete before merging.